pkg/controller/datavolume/conditions.go

```diff
@@ -134,11 +134,16 @@ func updateBoundCondition(conditions []cdiv1.DataVolumeCondition, pvc *corev1.Pe
 				conditions = UpdateReadyCondition(conditions, corev1.ConditionFalse, "", "")
 			}
 		case corev1.ClaimPending:
+			pvcPrimeName := getPrimeMessage(pvc)
```

Comment:
Is this function returning the prime name or "prime message"? what's exactly prime message? I think we can improve the naming a bit here.